Dockerfile

```diff
@@ -1,36 +1,36 @@
 # ------------------------------------------------------------------------------
 # Builder Stage
 # ------------------------------------------------------------------------------
-FROM golang:1.19-bullseye AS build
+FROM golang:1.19-alpine AS build
+
+RUN apk add --no-cache make
 
 WORKDIR /build
-ADD . /build
 
+COPY go.mod go.sum ./
 RUN go mod download
-RUN make build
+
+COPY Makefile main.go ./
+COPY static/ static/
+COPY pkg/ pkg/
+COPY .git/ .
+RUN CGO_ENABLED=0 make build
 
 # ------------------------------------------------------------------------------
-# Release Stage
+# Fetch wait-for
 # ------------------------------------------------------------------------------
-FROM debian:bullseye-slim
-
-RUN \
-  apt-get update && \
-  apt-get install -y ca-certificates openssl netcat curl gnupg lsb-release && \
-  update-ca-certificates
-
-RUN \
-  curl --silent https://www.postgresql.org/media/keys/ACCC4CF8.asc | apt-key add && \
-  echo "deb http://apt.postgresql.org/pub/repos/apt/ `lsb_release -cs`-pgdg main" | tee  /etc/apt/sources.list.d/pgdg.list && \
-  apt-get update && apt-get install -y postgresql-client
-
-RUN \
-  apt-get clean autoclean && \
-  apt-get autoremove --yes && \
-  rm -rf /var/lib/{apt,dpkg,cache,log}/
+FROM scratch AS wait
+ADD https://raw.githubusercontent.com/eficode/wait-for/v2.2.4/wait-for /usr/bin/wait-for
 
+# ------------------------------------------------------------------------------
+# Release Stage
+# ------------------------------------------------------------------------------
+FROM busybox:musl
```

Comment:
This needs to support current build targets as defined in here. https://github.com/sosedoff/pgweb/blob/master/.github/workflows/docker.yml#L35

As for having a very slim base image - there might be some other issues (like the missing postgres client package) and unfortunately I did not write down any notes since the last time I had to deal with this. A proper E2E test using Docker image is probably in order, however definitely out of scope of the PR.

Reply:
No problem then :)